cpp/src/parquet/parquet.thrift

```diff
@@ -585,15 +585,15 @@ union BloomFilterAlgorithm {
 /** Hash strategy type annotation. xxHash is an extremely fast non-cryptographic hash
  * algorithm. It uses 64 bits version of xxHash. 
  **/
-struct XxHash {}
+struct XxHasher {}
```

Comment:
Should we update the generated cop as well? And is it good time to sync the latest thrift file from upstream parquet-format? @wjones127 @pitrou

Reply:
Actually is this change necessary?

We could sync the file. It just has updated comments with clarifications and some deprecations.

Reply:
Double checked the upstream. This line should be reverted.